ruuvitag_sensor/adapters/bleak_ble.py

```diff
@@ -26,7 +26,11 @@ def _get_scanner(detection_callback):
 
         return DevBleakScanner(detection_callback, scanning_mode)
 
-    return BleakScanner(detection_callback=detection_callback, scanning_mode=scanning_mode)
+    if bt_device:
```

Comment:
Is there a more sensible way to do this than with a if-statement?

Reply:
When looking at the signature of `BleakScanner` (https://github.com/hbldh/bleak/blob/48b1963a7fa3fd23ccd6e1a919fad01907083fed/bleak/__init__.py#L134) maybe something like this will work? There is still an if-statement though 😆 

```py
bt_device="hc10"
additional_arguments={"adapter":bt_device} if bt_device else {}
return BleakScanner(detection_callback=detection_callback, scanning_mode=scanning_mode, kwargs=additional_arguments)
```

Reply:
There is but there is no second call to BleakScanner so I think your solution is better.